src/diffusers/schedulers/scheduling_ddpm.py

```diff
@@ -147,7 +147,7 @@ def __init__(
 
         self.variance_type = variance_type
 
-    def set_timesteps(self, num_inference_steps: int):
+    def set_timesteps(self, num_inference_steps: int, **kwargs):
```

Comment:
Here we want to avoid having unused `kwargs`  as function argument.

Reply:
It's to avoid having errors when calling `set_timesteps` with the device kwarg

Reply:
Should i remove it nonetheless?

Reply:
Big +1 let's not add a `kwarg` here